content/docs/guides/aws-lambda.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Connect from AWS Lambda
+enableTableOfContents: true
+---
+
+AWS Lambda is a serverless, event-driven compute service that allows you to run code without provisioning or managing servers. It is a convenient and cost-effective solution for running various types of workloads, including those that require a database.
+
+This guide describes how to set up a Neon database and connect to it from an AWS Lambda function using Node.js as the runtime environment. It covers:
+
+- Creating a Lambda function using the [Serverless Framework](https://www.serverless.com/), which is a serverless application lifecycle management framework.
+- Connecting your Lambda function to a Neon database.
+- Deploying the Lambda function to AWS.
+
+## Prerequisites
+
+- A Neon account. If you do not have one, see [Sign up](https://neon.tech/docs/get-started-with-neon/signing-up/) for instructions.
+- An AWS account. You can create a free AWS account at [AWS Free Tier](https://aws.amazon.com/free/).
+
+## Create a table in Neon
+
+To create a table, navigate to the **SQL Editor** in the [Neon Console](https://console.neon.tech/), or connect to your project using Neon's [passwordless connect](../../connect/passwordless-connect/) feature:
+
+```bash
+psql -h pg.neon.tech
+```
+
+In the SQL Editor or from the `psql` command-line, run the following queries to create a `users` table and insert some data:
+
+```sql
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    email TEXT NOT NULL,
+    created_at TIMESTAMPTZ DEFAULT NOW()
+);
+
+INSERT INTO users (name, email)
+VALUES
+    ('Alice', 'alice@example.com'),
+    ('Bob', 'bob@example.com'),
+    ('Charlie', 'charlie@example.com'),
+    ('Dave', 'dave@example.com'),
+    ('Eve', 'eve@example.com');
+```
+
+## Create a Lambda function
+
+The following steps describe how to create the Lambda function using the [Serverless Framework](https://www.serverless.com/).
+
+1. Install the Serverless Framework by running the following command:
+
+    ```bash
+    npm install -g serverless
+    ```
+
+2. Create a `my-lambda` project and change to that directory.
+
+    ```bash
+    mkdir my-lambda
+    cd my-lambda
+    ```
+
+3. Run the **serverless** command to create a serverless project. When prompted, select the **AWS - Node.js - Starter** project and provide your AWS account credentials, which include your AWS Access Key Id and AWS Secret Access Key. The process creates an `aws-node-project` directory.
```

Comment:
I thin here we should show the terminal output when running this command

Reply:
Added!